TLM/TLM/Util/RoundaboutMassEdit.cs

```diff
@@ -320,5 +350,25 @@ public class RoundaboutMassEdit {
             }
             return false;
         }
+
+        public void ClearNode(ushort nodeId) {
```

Comment:
`/// <summary>Removes lane connectors from all segments connected to this node.</summary>`

Reply:
it does much more than that. It clears all traffic rules related to roundabout. Note:
```
 PriorityRoad.ClearNode(nodeId);
```